tests/scripts/thread-cert/simulator.py

```diff
@@ -69,7 +69,7 @@ def set_lowpan_context(self, cid, prefix):
     def get_messages_sent_by(self, nodeid):
         raise NotImplementedError
 
-    def go(self, duration, nodeid=None):
+    def go(self, duration, nodeid=None, is_posix=False):
```

Comment:
I think the simulator already know if each node is posix or not. It can save this information in when `self.simulator.add_node(self)`. 
```suggestion
    def go(self, duration, nodeid=None):
```

Reply:
Good point. Thank you!👍